For some reason when the check for PyQt5 is present, the applications fail to
start with the error below. Before the Qt applications were split out, it was
noted in the weboob setup.py that the check does not work properly [1]. Not
sure why it was reintroduced after the split.

[1] https://git.weboob.org/weboob/weboob/blob/83e59081a4d30194d4d3eeb87c73a2067b7070c4/setup.py#L182

pkg_resources.DistributionNotFound: The 'PyQt5' distribution was not found and is required by weboob-qt

--- setup.cfg.orig	2021-01-01 05:18:36 UTC
+++ setup.cfg
@@ -75,7 +75,6 @@ classifiers = 
 python_requires = >= 3.5
 install_requires = 
 	weboob
-	PyQt5
 
 [options.entry_points]
 gui_scripts = 
